cloudbuild.yaml

```diff
@@ -40,4 +40,31 @@ steps:
             --provenance="mode=max" \
             .
 
+  - name: "gcr.io/cloud-builders/docker-mr"
+    id: "build-image"
+    waitFor: ["create-builder"]
+    entrypoint: "bash"
+    args:
+      - "-c"
+      - |
+        #!/bin/bash
+        set -euxo pipefail
+        docker buildx build \
+            --push \
+            --platform linux/amd64 \
+            --build-arg LAUNCHPAD_VERSION_SHA=$COMMIT_SHA \
+            -t $LOCATION-docker.pkg.dev/$PROJECT_ID/$REPO_NAME-mr/image:$COMMIT_SHA \
+            -t $LOCATION-docker.pkg.dev/$PROJECT_ID/$REPO_NAME-mr/image:$SHORT_SHA \
+            -t $LOCATION-docker.pkg.dev/$PROJECT_ID/$REPO_NAME-mr/image:nightly \
+            --label org.opencontainers.image.revision=$COMMIT_SHA \
+            --label org.opencontainers.image.version=$COMMIT_SHA \
+            --label org.opencontainers.image.title=$REPO_NAME \
+            --label org.opencontainers.image.source="https://github.com/$REPO_FULL_NAME" \
+            --label org.opencontainers.vendor="Sentry" \
+            --label org.opencontainers.image.url=$REPO_FULL_NAME \
+            --cache-from $LOCATION-docker.pkg.dev/$PROJECT_ID/$REPO_NAME-mr/image:latest \
+            --sbom true \
+            --provenance="mode=max" \
+            .
```

Comment:
Up to you, but you don't have to duplicate this whole thing, you can add extra tags with `-t` to the existing `docker buildx build` job.